openrarity/io/read.py

```diff
@@ -0,0 +1,13 @@
+import json
+from pathlib import Path
+from typing import Any
+
+
+def from_json(
+    path: str | Path,
+) -> dict[str | int, dict[Any, Any]]:
+    return json.loads(Path(path).read_text())
+
+
+def from_csv(path: str | Path) -> dict[str | int, dict[Any, Any]]:
```

Comment:
perhaps remove it? why keep not implemented method?

Reply:
Was thinking it would be implemented eventually but honestly now I really don't intend to.